app/controllers/skateparks/searches_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Skateparks
+  class SearchesController < ApplicationController
+    def create
+      render partial: 'search_results', locals: {
+        skateparks: Skatepark.search(params[:search].downcase) }
+    end
+  end
+end
```

Comment:
I'm with that I'll hook it up @yago580. It kinda makes sense when you think about it as creating a search but yeah, in the context of the db, which is pretty much the general context of `#create`, it don't make sense.